tests/TestableIO.System.IO.Abstractions.TestingHelpers.Tests/MockDriveInfoTests.cs

```diff
@@ -84,5 +84,124 @@ public void MockDriveInfo_ToString_ShouldReturnTheDrivePath(string path, string
             // Assert
             Assert.That(mockDriveInfo.ToString(), Is.EqualTo(expectedPath));
         }
+
+        [Test]
+        public void MockDriveInfo_AvailableFreeSpace_ShouldReturnAvailableFreeSpaceOfDriveInMemoryFileSystem()
+        {
+            // Arrange
+            var availableFreeSpace = 1024L;
+            var driveData = new MockDriveData { AvailableFreeSpace = availableFreeSpace };
+            var fileSystem = new MockFileSystem();
+            fileSystem.AddDrive("C:", driveData);
+            var mockDriveInfo = new MockDriveInfo(fileSystem, "C:");
```

Comment:
I think a better test would be to get the `IDriveInfo` from the `fileSystem` and compare its values, e.g.
```csharp
var driveInfo = fileSystem.DriveInfo
    .GetDrives()
    .Single(x => x.Name == "C:\\");

// Act
var result = driveInfo.AvailableFreeSpace;
```

Reply:
Good point, changed it accordingly.